app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -307,7 +307,16 @@ public String findThumbnailByFilename(String filename) throws IOException {
     }
 
     @Override
-    public String parseWikicode(String source) throws IOException {
+    public String readFeedback(String source) throws IOException {
+        return wikidataApi.action("query")
+                .param("prop", "revisions")
+                .param("rvprop", "content")
+                .param("rvslots", "main")
+                .param("titles", "Talk:Q11311478")
```

Comment:
I guess implementing this is the next step? :-)

Reply:
@nicolas-raoul I have implemented this already. The text visible in the screenshot is the text fetched using this API call

Reply:
I mean, `Q11311478` is hard-coded right now, it needs to be replaced with the QID of the Wikidata item being viewed by the user :-)

Reply:
oh yes!!
@nicolas-raoul `getWikiDataEntityId` does this returns the same QID that we are searching for?

Reply:
Yes `Place:getWikiDataEntityId`